### PR TITLE
[EDMTP-195] Add the ability to set properties of Alert widget

### PR DESCRIPTION
**Pull request by liubou-masiuk**
- Provide 'set' and 'get' accessors for Alert widget
- Add new reference type: JSON
- Provide the ability to set serialized objects in text and hidden fields
- Add objects comparison to `isEqual` method
- Update Readme

### CODE REVIEW

File: aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/accessors/dependsOnCoralAlertAccessor.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * */
+(function ($, ns) {
+    const ALERT_SELECTOR = '.coral3-Alert';
+
+    ns.ElementAccessors.registerAccessor({
+        selector: `${ALERT_SELECTOR}`,
+        preferableType: 'object',
+        get: function ($el) {
+            const element = $el[0];
+            let result = {}
```
Comment (ala-n) [inline aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/accessors/dependsOnCoralAlertAccessor.js:15]: const

File: aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/accessors/dependsOnJsonSetAccessor.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/**
```
Comment (ala-n) [inline aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/accessors/dependsOnJsonSetAccessor.js:1]: Super cosmetic
but not sure that we need to consider it as something special, it was planed as mostly basic accessor
Mb we need to consider it a default behaviour for object input.

File: aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/dependsOnUtils.js [hunk 3]
```diff
@@ -54,6 +60,12 @@
                 return Number(value);
             case 'string':
                 return String(value);
+            case 'json':
+                try {
```
Comment (ala-n) [inline aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/dependsOnUtils.js:64]: Cosmetic: move to Utilities.parseSafe

**Comment by ala-n** [comment]
Agree with @smiakchilo 
That looks really nice!

**Comment by ala-n** [comment]
Same question her
@smiakchilo, @liubou-masiuk 
Can we start merging PRs without technical concern to get actual target state for other related changes?

File: aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/accessors/dependsOnJsonSetAccessor.js (hunk 2 above)
Comment (liubou-masiuk) [reply aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/accessors/dependsOnJsonSetAccessor.js]: moved to the base 'set' accessor

File: aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/accessors/dependsOnCoralAlertAccessor.js (hunk 1 above)
Comment (liubou-masiuk) [reply aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/accessors/dependsOnCoralAlertAccessor.js]: fixed, thank you!

File: aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/dependsOnUtils.js (hunk 3 above)
Comment (liubou-masiuk) [reply aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/dependsOnUtils.js]: done

**Pull request merged**